docs/contributing/docs-templates.md

```diff
@@ -3,69 +3,82 @@ title: Docs Templates
 ---
 
 1.  [Why use templates?](#why-use-templates)
-2.  [Guide](#guide)
-    - [Guide template](#guide-template)
-3.  [Guide overview](#guide-overview)
-    - [Guide overview template](#guide-overview-template)
-4.  [Tutorial](#tutorial)
+2.  [Reference guides](#reference-guides)
+    - [Reference guide template](#reference-guide-template)
+    - [Reference guide overview template](#reference-guide-overview-template)
+3.  [Recipes](#recipes)
+    - [Recipe template](#recipe-template)
+4.  [Tutorials](#tutorials)
     - [Tutorial template](#tutorial-template)
 5.  [Plugin README template](#plugin-readme-template)
 6.  [Starter README template](#starter-readme-template)
 
-## The Gatsby Way(TM) of writing guides
+## The Gatsby Way of writing learning materials
+
+Docs guides, recipes, and tutorials are intended to facilitate learning about Gatsby from a variety of perspectives and skill-sets.
```

Comment:
are docs guides the same as reference guides?

Reply:
Yes! They are the same. I can make the references more consistent.

Reply:
After reading the section above about blog posts/reference guides/recipes/tutorials it sort of feels like "Docs" is an umbrella term that encompasses all 4 of those, and these terms are just more specific subsets of "Docs"? Is that a correct understanding?

Reply:
Yes, I'd say that's correct! Tutorials are in their own section for discoverability, but conceptually I would include them in "Docs".